htmy/tag.py

```
@@ -0,0 +1,101 @@
+
+    __slots__ = ("name",)
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def __call__(self, **props: PropertyValue) -> ComponentType:
+        return _TagWithPropsImpl(self.name, props)
+
+
+class Tag:
+    __slots__ = ("name", "child_separator")
```

Ruff (0.14.7)

`Tag.__slots__` is not sorted

Apply a natural sort to `Tag.__slots__`

(RUF023)